lisp/ess-r-syntax.el

```diff
@@ -358,8 +358,7 @@ reached."
       (ess-token-after= '("," ";"))
       (and (ess-token-after= "identifier")
            (not (memq (char-syntax (char-before)) '(?w ?_))))
-      (progn (/= (skip-syntax-backward ".") 0)
-             (ess-token-operator-p (ess-token-after)))
+      (ess-token-operator-p (ess-token-after))
```

Comment:
What is the purpose here? I don't see a compilation warning.

Reply:
Without this change, I see:

```
ess-r-syntax.el:346:8:Warning: value returned from (/= (skip-syntax-backward
                                                        ".") 0) is unused
```

Reply:
oops of course, I wonder why I didn't see any warning.

Could you keep the `skip-syntax-backward` though? Not sure if it is needed (no tests seem to have broken) but better be safe.

Reply:
Done